src/modules/fuzzy-search/look-up.model.ts

```diff
@@ -0,0 +1,18 @@
+import { Column, PrimaryKey, Table, Model } from 'sequelize-typescript';
+import { LookUpAttributes } from './look-up.domain';
+
+@Table({
+  underscored: true,
+  tableName: 'look_up',
+})
+export class LookUpModel extends Model implements LookUpAttributes {
+  @PrimaryKey
+  @Column
+  id: string;
+
+  @Column
+  userUuid: string;
```

Comment:
Reference User['uuid'] field as the type instead, please. Also let's call the userUuid column as userId as we are going to deprecate the auto incremental id at some point